Lsof.8

```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, lsof-org team at GitHUB maintains.
```

Comment:
still would like to see:

Since version 4.93.0, the lsof-org team at GitHub maintains lsof.